services/galley/src/Galley/API/Teams.hs

```diff
@@ -172,6 +173,8 @@ createBindingTeam zusr tid (BindingNewTeam body) = do
   let owner = newTeamMember zusr fullPermissions Nothing
   team <- Data.createTeam (Just tid) zusr (body ^. newTeamName) (body ^. newTeamIcon) (body ^. newTeamIconKey) Binding
   finishCreateTeam team owner [] Nothing
+  Data.addBillingTeamMember tid (owner ^. userId)
```

Comment:
Can't we instead do this inside the data model when adding members so we can do this transactionally?

Reply:
We don't want to do this for NonBindingTeams, so we'll have to put a check there. I am also not sure how we decide what goes in `API.Teams` and what goes in `Galley.Data`. Can you please elaborate on the reasoning?

Reply:
I think it's fine to do it for `BindingTeams` as members of those teams also have the `SetBilling` permission